Source/Game.cs

```diff
@@ -185,6 +224,44 @@ public Game()
 	#endregion
 
 	#region Methods
+	/// <summary>
+	/// Request a full save of all persistence files.
+	/// </summary>
+	public static void RequestSave()
```

Comment:
Replace with this?
```c#
public static async Task RequestSave() {
    // This three lines only prevent queuing saves (since that may be redundant)
    bool status = false;
    Monitor.TryEnter(savelock, 0/* ms timeout */, ref status);
    if (status) return;

    lock (savelock) {
        Save.SaveToFile();
        Settings.SaveToFile();
        Controls.SaveToFile();
        ModSettings.SaveToFile();
    }
}
```
And clean up all the now unused fields ofc